src/nyc_trees/apps/core/templates/skeleton.html

```diff
@@ -15,6 +15,7 @@
         <link rel="stylesheet" type="text/css" href="//cloud.typography.com/7429932/717646/css/fonts.css" />
     </head>
     <body>
+    {% block modals %}{% endblock modals %}
```

Comment:
We've been using the existing `extra_content` block for modals.
https://github.com/azavea/nyc-trees/blob/develop/src/nyc_trees/apps/core/templates/base.html#L64